app/src/main/java/com/team41/boromi/BookActivity.java

```diff
@@ -0,0 +1,105 @@
+package com.team41.boromi;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
+
+import com.team41.boromi.book.OwnedFragment;
+import com.team41.boromi.book.BorrowedFragment;
+import com.team41.boromi.book.MapFragment;
+import com.team41.boromi.book.SearchFragment;
+import com.team41.boromi.book.SettingsFragment;
+
+public class BookActivity extends AppCompatActivity {
+
+  Button bookButton;
+  Button borrowedButton;
+  Button searchButton;
+  Button mapButton;
+  Button settingsButton;
```

Comment:
A couple of questions for anyone who wants to answer
1. Should component attributes like this be public?
2. Should public attributes/methods be declared public implicitly or explicitly?